File: src/exercises/hello.f90
program hello

  include 'mpif.h'

  integer my_rank, my_size, errcode

  call MPI_INIT(errcode)

  call MPI_COMM_RANK(MPI_COMM_WORLD, my_rank, errcode)
  call MPI_COMM_SIZE(MPI_COMM_WORLD, my_size, errcode)
  print *, 'Hello from ', my_rank, 'of', my_size, '.'

  call MPI_FINALIZE(errcode)

end program hello

File: src/exercises/hello.c
//#include <stdio.h>
#include "mpi.h"

main(int argc, char** argv){

  int my_rank, my_size;

  MPI_Init(&argc, &argv);

  MPI_Comm_rank(MPI_COMM_WORLD, &my_rank);
  MPI_Comm_size(MPI_COMM_WORLD, &my_size);
  printf("Hello from %d of %d.\n", my_rank, my_size);

  MPI_Finalize();

}
